File: python/src/unittest/python/TortillaMaiz_tests.py
from TortillaMaiz import *
import unittest

class TortillaMaizTest(unittest.TestCase):
    def setUp(self):
        self.tortillaMaiz = TortillaMaiz()

    def test_CurrentTemperature(self):
        self.tortillaMaiz.setCurrentTemperature(21)
        self.assertEquals(21,self.tortillaMaiz.getCurrentTemperature())

    def test_FalseToast(self):
        self.tortillaMaiz.toast(False)
        self.assertFalse(self.tortillaMaiz.isToasted())

    def test_TrueToast(self):
        self.tortillaMaiz.toast(True)
        self.assertTrue(self.tortillaMaiz.isToasted())

    def test_Toasting(self):
        self.assertEquals(10,self.tortillaMaiz.getToastTemperature())

File: csharp/unittest-practiceTests/Clases/TortillaMaizTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using unittestpractice.Clases;

namespace unittest_practiceTests.Clases
{
    [TestClass]
    public class TortillaMaizTests
    {
        //Asserts
        private readonly TortillaMaiz _tortillaMaiz = new TortillaMaiz();

        [TestMethod]
        public void TestCurrentTemperature()
        {
            _tortillaMaiz.SetCurrentTemperature(21);
            Assert.AreEqual(21, _tortillaMaiz.GetCurrentTemperature());
        }

        [TestMethod]
        public void TestFalseToast()
        {
            _tortillaMaiz.Toast(false);
            Assert.IsFalse(_tortillaMaiz.IsToasted());
        }

        [TestMethod]
        public void TestTrueToast()
        {
            _tortillaMaiz.Toast(true);
            Assert.IsTrue(_tortillaMaiz.IsToasted());
        }

        [TestMethod]
        public void TestToasting()
        {
            Assert.AreEqual(35, _tortillaMaiz.GetToastTemperature());
        }
    }
}
